errbot/errBot.py

```diff
@@ -265,6 +266,18 @@ def process_message(self, mess):
     def _process_command(self, mess, cmd, args, match):
         """Process and execute a bot command"""
 
+        # first it must go through the command filters
+        try:
+            for cmd_filter in self.command_filters:
+                mess, cmd, args = cmd_filter(mess, cmd, args)
+                if mess is None:
+                    log.info("Command %s deferred." % cmd)
+                    return
+        except ACLViolation as e:
```

Comment:
This tightly couples `errBot` itself with the ACL filter, which I think we should try to avoid to avoid *"special casing"* certain filters.

I wonder if we can make this more generic. Have filters signal whether the bot should continue processing the command (having it go through other filters, if any, then run the command) or if it should stop processing (because the filter blocked it)